backend/app/users/types.py

```diff
@@ -75,6 +75,12 @@ def from_document(doc: Mapping[str, any]) -> "UserPreferences":
     class Config:
         extra = "forbid"
 
+class GetUsersPreferencesResponse(UserPreferences):
+    sessions_with_feedback: list[int] = Field(default_factory=list)
```

Comment:
we should return a boolean here that is true if the user has submitted feedback for this session

Reply:
feedback_submitted: true/false